bamt/utils/EvoUtils.py

```diff
@@ -0,0 +1,113 @@
+import random
+
+import pandas as pd
+
+from pgmpy.estimators import K2Score
+from pgmpy.models import BayesianNetwork
+from golem.core.dag.convert import graph_structure_as_nx_graph
+from golem.core.dag.graph_utils import ordered_subnodes_hierarchy
+from golem.core.optimisers.graph import OptGraph, OptNode
+
+
+class CustomGraphModel(OptGraph):
+    def evaluate(self, data: pd.DataFrame):
+        nodes = data.columns.to_list()
+        _, labels = graph_structure_as_nx_graph(self)
+        return len(nodes)
```

Comment:
Why do we need this method?

Reply:
not sure, but it was there in  GOLEM examples, we might need it in the future